<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="fraSpectrum">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>861</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Spectrum</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Graphplot" name="SpectrumPlot" native="true">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="fraDetector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Detector</string>
   </property>
   <widget class="QPushButton" name="cmdDetector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraSettings">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>MCA settings</string>
   </property>
   <widget class="QPushButton" name="cmdSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraTiming">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Timing</string>
   </property>
   <widget class="QLabel" name="lblRealtime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Realtime</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRealtime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLivetime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Livetime</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLivetime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraControls">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Detector</string>
   </property>
   <widget class="QPushButton" name="cmdStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdStop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdClear">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Graphplot</class>
   <extends>QWidget</extends>
   <header>graphplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdDetector</tabstop>
  <tabstop>cmdSettings</tabstop>
  <tabstop>txtLivetime</tabstop>
  <tabstop>txtRealtime</tabstop>
  <tabstop>cmdStart</tabstop>
  <tabstop>cmdStop</tabstop>
  <tabstop>cmdClear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
